python/tvm/relay/op/strategy/arm_cpu.py

```diff
@@ -242,7 +242,14 @@ def conv2d_strategy_arm_cpu(attrs, inputs, out_type, target):
                             ),
                             name="conv2d_NHWC_quantized_interleaved.arm_cpu",
                         )
-                if (not is_aarch64) or (data.dtype not in ["int8", "uint8"]):
+                # Non-quantized cases
+                if is_aarch64 and data.dtype in ["float32", "float16"]:
+                    strategy.add_implementation(
+                        wrap_compute_conv2d(topi.arm_cpu.compute_conv2d_NHWC_hybrid),
+                        wrap_topi_schedule(topi.arm_cpu.schedule_conv2d_NHWC_hybrid),
+                        name="conv2d_NHWC_hybrid.arm_cpu",
+                    )
+                if (not is_aarch64) or (data.dtype not in ["int8", "uint8", "float32", "float16"]):
```

Comment:
Apologies for missing this before, it might actually be better to keep the original condition here: `if not is_aarch64 or data.dtype not in ["int8", "uint8"]` and use `add_implementation`'s `plevel` argument to prioritise the hybrid schedule. This way if autoTVM is being used, both schedule options are exposed for selection.

Reply:
That makes sense, I'll make the change.